baseapp/baseapp.go

```diff
@@ -484,16 +484,18 @@ func (app *BaseApp) runTx(mode runTxMode, txBytes []byte, tx sdk.Tx) (result sdk
 	}()
 
 	// Get the Msg.
-	var msg = tx.GetMsg()
-	if msg == nil {
+	var msgs = tx.GetMsgs()
+	if msgs == nil {
```

Comment:
or `len(msgs) == 0`, I think, no reason to allow transactions with zero messages